.pipelines/js-ci.yml

```diff
@@ -0,0 +1,84 @@
+name: $(date:yyMM).$(date:dd)$(rev:rrr)
+
+pr:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - source/nodejs
+
+pool:
+  name: Azure Pipelines
+  vmImage: windows-2019
+  demands:
+  - npm
+
+parameters:
+- name: target_packages
+  displayName: "Which packages is going to be built."
+  type: object
+  default:
+  - adaptivecards
+  - adaptivecards-controls
+  - adaptivecards-designer
+  - adaptivecards-templating
+  - adaptivecards-react
+
+- name: test_projects
+  displayName: "Which project is going to be consume the built packages."
+  type: object
+  default:
+  - adaptivecards-ui-testapp
+  - adaptivecards-react-testapp
+
+steps:
+- task: NodeTool@0
+  name: NodeTool1
+  displayName: Use Node 14.x
+  inputs:
+    versionSpec: 14.x
+
+- bash: |
+   npm i -g npm@latest
+   npm ci 
+   npx lerna bootstrap --ci
+   npx lerna run release
+  workingDirectory: source/nodejs
+  displayName: 'Bash - lerna bootstrap'
+
+- bash: |
```

Comment:
the tests at source/nodejs/tests/ui-tests are not being run

Reply:
build-and-test all project in /tests/